src/stories/Forms/Form/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const FormElement = styled.form`
+  /* Your styles here */
```

Comment:
Can this be removed if it is empty?

Reply:
Its just a placeholder for now I guess as I don't have any final design on it afaik.